src/components/transport_manager/src/usb/libusb/usb_handler.cc

```diff
@@ -90,20 +90,24 @@ UsbHandler::UsbHandler()
 
 UsbHandler::~UsbHandler() {
   shutdown_requested_ = true;
-  if (libusb_context_ != 0) {
+  thread_->join(threads::Thread::kForceStop);
+  delete thread_->delegate();
+  threads::DeleteThread(thread_);
+  SDL_INFO("UsbHandler thread finished");
+
+  if (libusb_context_ != NULL) {
     libusb_hotplug_deregister_callback(libusb_context_,
                                        arrived_callback_handle_);
     libusb_hotplug_deregister_callback(libusb_context_, left_callback_handle_);
-  }
-  thread_->stop();
-  SDL_INFO("UsbHandler thread finished");
-  if (libusb_context_) {
+
+// TODO(VlAntonov) APPLINK-27938
+// This fix prevents crash under Windows but causes memory leaks
```

Comment:
@anosach-luxoft Inside the `libusbx` there is an access to invalid data. See details in [APPLINK-27938](https://adc.luxoft.com/jira/browse/APPLINK-27938)